soc/microchip/pic32cxsg/common/Kconfig.pic32cxsg

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Microchip
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_PIC32CXSG41 || SOC_SERIES_PIC32CXSG60 || SOC_SERIES_PIC32CXSG61
```

Comment:
If the whole family covers then it is better to use CONFIG_SOC_FAMILY_MICROCHIP_PIC32CXSG.

Reply:
Thanks for pointing this out. Resolved.